app/src/main/java/com/movieapp/viewmodel/MovieListViewModel.java

```diff
@@ -0,0 +1,43 @@
+package com.movieapp.viewmodel;
+
+import android.app.Application;
+
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import com.movieapp.model.Movie;
+import com.movieapp.model.TopRatedMoviesResponse;
+import com.movieapp.network.MovieDatabaseAPI;
+import com.movieapp.network.NetworkAPI;
+import com.movieapp.repository.MovieRepository;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class MovieListViewModel extends AndroidViewModel {
```

Comment:
Why do you extend from AndroidViewModel?

Reply:
Fixed